apps/web/components/Organization/NewsArticle/NewsArticle.tsx

```
@@ -150,6 +152,16 @@ export const NewsArticle: React.FC<
         newsItem?.fullWidthImageInContent && (
           <NewsItemImage newsItem={newsItem} />
         )}
+
+      {bottomSlices.length > 0 && (
+        <Box paddingTop={3}>
+          <Stack space={3}>
+            {bottomSlices.map((slice) => (
+              <EmailSignup slice={slice} />
+            ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)